settings.py

```diff
@@ -23,6 +23,10 @@
 
 USE_TZ = False
```

Comment:
Consider setting `USE_TZ = True` to enable timezone-aware datetimes, which is recommended for most Django projects to handle timezones correctly.